[2024-09-03T06:31:52.500+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-03T06:31:52.519+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v2.insert_into_table scheduled__2024-09-02T00:00:00+00:00 [queued]>
[2024-09-03T06:31:52.528+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v2.insert_into_table scheduled__2024-09-02T00:00:00+00:00 [queued]>
[2024-09-03T06:31:52.529+0000] {taskinstance.py:2856} INFO - Starting attempt 3 of 8
[2024-09-03T06:31:52.545+0000] {taskinstance.py:2879} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2024-09-02 00:00:00+00:00
[2024-09-03T06:31:52.554+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v2', 'insert_into_table', 'scheduled__2024-09-02T00:00:00+00:00', '--job-id', '461', '--raw', '--subdir', 'DAGS_FOLDER/postgres_operator_dag.py', '--cfg-path', '/tmp/tmpr42ud2ut']
[2024-09-03T06:31:52.557+0000] {standard_task_runner.py:105} INFO - Job 461: Subtask insert_into_table
[2024-09-03T06:31:52.557+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=22893) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-03T06:31:52.557+0000] {standard_task_runner.py:72} INFO - Started process 22894 to run task
[2024-09-03T06:31:52.601+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_postgres_operator_v2.insert_into_table scheduled__2024-09-02T00:00:00+00:00 [running]> on host efc48c282cbe
[2024-09-03T06:31:52.689+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ubeyd' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v2' AIRFLOW_CTX_TASK_ID='insert_into_table' AIRFLOW_CTX_EXECUTION_DATE='2024-09-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-02T00:00:00+00:00'
[2024-09-03T06:31:52.690+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-03T06:31:52.703+0000] {sql.py:282} INFO - Executing: 
            insert into dag_runs (dt, dag_id) values ('2024-09-02', 'dag_with_postgres_operator_v2')
        
[2024-09-03T06:31:52.713+0000] {base.py:84} INFO - Retrieving connection 'postgres_localhost'
[2024-09-03T06:31:52.720+0000] {base.py:84} INFO - Retrieving connection 'postgres_localhost'
[2024-09-03T06:31:52.729+0000] {sql.py:509} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2024-09-02', 'dag_with_postgres_operator_v2')
        , parameters: None
[2024-09-03T06:31:52.732+0000] {sql.py:518} INFO - Rows affected: 1
[2024-09-03T06:31:52.750+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-03T06:31:52.750+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v2, task_id=insert_into_table, run_id=scheduled__2024-09-02T00:00:00+00:00, execution_date=20240902T000000, start_date=20240903T063152, end_date=20240903T063152
[2024-09-03T06:31:52.773+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-09-03T06:31:52.789+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-03T06:31:52.795+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
